### DISCUSSION TITLE
[RFC] Spin 组件支持设定背景透明度

### DISCUSSION THREAD

**Discussion by lcgash** [ant-design/ant-design#47827] [RFCs]
## Summary

Spin 组件在我们的场景下，有需要定制修改背景透明度的需求。他也并不是简单的作为加载动画使用，有时会去承担禁用解封之类的状态，这种场景下是有必要去模糊或者更清晰背景的需求。

### 方案 1
增加属性，内置三种透明度
![image](https://github.com/ant-design/ant-design/assets/37072324/5bfca8e6-7717-4bc5-9038-8b9941b7bf6c)
不同的属性代表了不同的背景透明度
![image](https://github.com/ant-design/ant-design/assets/37072324/14d35ee6-a5e2-493a-9f4f-da5a26cfbd06)
效果图
![image](https://github.com/ant-design/ant-design/assets/37072324/4f503ea7-f9c1-4408-890d-02512aa294f1)

### 方案 2
通过 blurOpacity 的 token 暴露出去，用户完全自定义透明度的大小。

**Comment by Wxh16144** [comment]
个人偏向后者，完全由开发者决定透明度。

**Comment by li-jia-nan** [comment]
我觉得两个方案可以同时支持

**Comment by MadCcc** [comment]
方案2️⃣：提供 `contentOpacity` 组件 token
